packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -87,6 +87,11 @@ interface SupportedVsCurrenciesCache {
   data: string[];
 }
 
+enum PollState {
```

Comment:
Good idea to use an enum for the poll state. Do you envision adding other states in the future, such as "paused" or something like that?

Reply:
I was thinking of an error state, if we notice it failing continuously for long enough. Not sure though